.github/workflows/slow-tests.yml

```diff
@@ -0,0 +1,62 @@
+# A weekly workflow to run slow tests with MLFLOW_RUN_SLOW_TESTS environment variable set to true.
+name: MLflow Slow Tests
+
+on:
+  workflow_dispatch:
```

Comment:
Can we run this workflow when this file (.github/workflows/slow-tests.yml) is changed?